locator/topology.cc

```diff
@@ -580,20 +580,31 @@ void topology::sort_by_proximity(locator::host_id address, host_id_vector_replic
 
 template <typename T>
 void topology::do_sort_by_proximity(T address, utils::small_vector<T, 3>& addresses) const {
```

Comment:
2.8k cycles is much better than 13k cycles, but still outrageous.

What are we measuring here?

Reply:
sort_by_proximity of 15 nodes using a reference node that is one of them